### ISSUE TITLE
Console warning in /contents view

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1695]
```
Warning: Failed prop type: Invalid prop `size` of value `none` supplied to `Confirm`, expected one of ["mini","tiny","small","large","fullscreen"].
    in Confirm (at Contents.jsx:977)
```

This should be easy to fix.

**Comment by silviubogan** [comment]
Solved in the PR #1702.

**Comment by nileshgulia1** [comment]
Thanks @silviubogan
